draft-ietf-jsonpath-iregexp.md

```diff
@@ -218,10 +218,10 @@ exceptions:
 
 * Not all regexp implementations
   support accesses to Unicode tables that enable
-  executing on constructs such as `\p{IsCoptic}`,
+  executing on constructs such as `\p{Nd}`,
   although the `\p`/`\P` feature in general is now quite
   widely available. While in principle it’s possible to
-  translate these into codepoint-range matches, this also requires
+  translate these into character-class matches, this also requires
   access to those tables. Thus, regexp libraries in severely
   constrained environments may not be able to support I-Regexp
```

Comment:
Is this still true with the removal of unicode blocks?

Reply:
Yes.  Turns out that my homework assignment proving this was done already:

https://github.com/CESNET/libyang/blob/87375f15159545a87a1e0de200f5d9d67e9091d7/src/schema_compile_node.c#L1023

This is a bit icky, as we don't have a \x or \u in iregexp, so you would have to include the actual unicode characters.